### ISSUE TITLE
Error: HTTP Error: 400, Request contains an invalid argument.

### ISSUE THREAD

**Issue by narita1980** [firebase/friendlyeats-web#56]
https://codelabs.developers.google.com/codelabs/firestore-web/#4

An error will occur if you execute the following command.

> Run the following Firebase CLI command:
> firebase serve

The errors that occur are as follows.
```
$ firebase serve

=== Serving from '・・・/friendlyeats-web'...

✔  firestore: started on http://localhost:8080

Error: HTTP Error: 400, Request contains an invalid argument.
⚠  firestore: Nov 16, 2018 3:43:44 AM io.gapi.emulators.netty.NettyUtil applyJava7LongHostnameWorkaround
INFO: Unable to apply Java 7 long hostname workaround.

$
```

Do you have any solutions?

P.S.
```
$ uname -a
Darwin local 17.4.0 Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64 x86_64

$ firebase --version
6.1.0
```

**Comment by Yatima-Kagurazaka** [comment]
How about  `firebase serve --only hosting` ?

**Comment by morganchen12** [comment]
Fixed in #57.
